pkgs/development/haskell-modules/configuration-ghc-7.10.x.nix

```diff
@@ -80,6 +91,30 @@ self: super: {
   # Test suite fails in "/tokens_bytestring_unicode.g.bin".
   alex = dontCheck super.alex;
 
+  haddock-library = overrideCabal super.haddock-library (drv: {
```

Comment:
> I'm not sure if it is worth adding workarounds for the expected two or three weeks if everything goes according to plan

@cocreature: I agree. I just wanted to get everything building with Haskell-NG and get an early review of the basic approach. I think we should just wait for the next GHC code drop, and then I'll go back and remove the workarounds.